pytorch_lightning/plugins/training_type/ipu.py

```diff
@@ -120,6 +120,16 @@ def setup(self) -> None:
         self._update_dataloader_original = pl.trainer.data_loading._update_dataloader
         pl.trainer.data_loading._update_dataloader = self._convert_to_poptorch_loader
 
+        if not self.setup_optimizers_in_pre_dispatch:
```

Comment:
We don't need `setup_optimizers_in_pre_dispatch` now. Maybe next PR ?

Reply:
Filed issue https://github.com/PyTorchLightning/pytorch-lightning/issues/10685